src/content/docs/de/guides/troubleshooting.mdx

```diff
@@ -0,0 +1,208 @@
+---
+title: Fehlerbehebung
+description: Brauchst du Hilfe? Hängst du bei etwas fest? Wir haben das Richtige für dich.
+i18nReady: true
+---
+
+Astro bietet verschiedene Tools, die dir bei der Fehlersuche und beim Debuggen deines Codes helfen.
+
+## Allgemeine Fehlermeldungen
+
+Hier eine Auswahl häufiger Fehlermeldungen, die im Terminal erscheinen können, was sie bedeuten können und was du dagegen tun kannst.
+
+### Cannot use import statement outside a module
+
+In Astro-Komponenten werden `<script>`-Tags standardmäßig als [JS-Module](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) gehoistet und geladen. Wenn du die Direktive [`is:inline`](/de/reference/directives-reference/#isinline) oder ein anderes Attribut in deinen Tag eingebaut hast, wird dieses Standardverhalten aufgehoben.
```

Comment:
Ich wuerde "gehoisted" hier durch ein deutsches Wort ersetzen, Was mir dazu einfallen wurde ist "erstellt" oder "festeghalten"
```suggestion
In Astro-Komponenten werden `<script>`-Tags standardmäßig als [JS-Module](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) festeghalten und geladen. Wenn du die Direktive [`is:inline`](/de/reference/directives-reference/#isinline) oder ein anderes Attribut in deinen Tag eingebaut hast, wird dieses Standardverhalten aufgehoben.
```

Reply:
Es gibt leider keine sinnvolle deutsche Entsprechung dafür, zumindest nicht in einem Wort. Wir verwenden das intern auch so, kenne auch niemanden, der es in diesem Kontext nicht so verwendet, würde es daher so lassen.